itest/lnd_channel_backup_test.go

```diff
@@ -106,13 +106,6 @@ func newChanRestoreScenario(ht *lntest.HarnessTest, ct lnrpc.CommitmentType,
 	// with a portion pushed.
 	ht.ConnectNodes(dave, carol)
 
-	// If the commitment type is taproot, then the channel must also be
```

Comment:
Out of place for this commit, but: I think we're missing a set of `lncfg` flags to enable/disable the new gossip.